src/main/java/com/pinterest/secor/io/impl/DelimitedTextFileReaderWriterFactory.java

```diff
@@ -58,76 +59,92 @@ public FileWriter BuildFileWriter(LogFilePath logFilePath, CompressionCodec code
     }
 
     protected class DelimitedTextFileReader implements FileReader {
-        private final BufferedInputStream mReader;
-        private long mOffset;
+        private final SequenceFile.Reader mOffsetReader;
+        private final LongWritable mKey;
+        private final IntWritable mValueLength;
+        private final DataInputStream mReader;
         private Decompressor mDecompressor = null;
+        private byte[] mDelimiter;
 
-        public DelimitedTextFileReader(LogFilePath path, CompressionCodec codec) throws IOException {
+        public DelimitedTextFileReader(LogFilePath path, CompressionCodec codec) throws Exception {
+            Configuration config = new Configuration();
```

Comment:
Reader is only used for trimming purpose and secor does not perform any recovering of existing file. So I don't see how this can break any existing installation.